packages/shared/src/components/auth/EmailCodeVerification.tsx

```diff
@@ -28,6 +36,7 @@ function EmailCodeVerification({
   onSubmit,
   className,
 }: EmailCodeVerificationProps): ReactElement {
+  const onboardingPapercuts = useFeature(featureOnboardingPapercuts);
```

Comment:
Also this gets used when retro-adding email afaik.

Reply:
The current focus of introducing this new component is for the onboarding registration. I think we can propagate the usage of this component if this introduction wins. Wdyt?

Reply:
Yeah it's fine, just enrollment might be bigger depending how you want to evaluate experiment win or not.